src/components/tabs/tabs.tsx

```diff
@@ -45,7 +38,7 @@ export type EuiTabsProps = CommonProps &
      * Sizes affect both font size and overall size.
      * Only use the `xl` size when displayed as page titles.
      */
-    size?: EuiTabsSizes;
+    size?: 's' | 'm' | 'l' | 'xl';
```

Comment:
Instead of deleting the type `EuiTabsSizes` you can keep it and modify it to look like this:

``` js
export const SIZES = ['s', 'm', 'l', 'xl'] as const;
export type EuiTabsSizes = typeof SIZES[number];
```

```suggestion
    size?: EuiTabsSizes;
```